kube/services/requestor/requestor-deploy.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: requestor-deployment
+spec:
+  selector:
+    # Only select pods based on the 'app' label
+    matchLabels:
+      app: requestor
+      release: production
+  revisionHistoryLimit: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: requestor
+        release: production
+        public: "yes"
+        # for network policy
+        netnolimit: "yes" # TODO do i need this?
```

Comment:
@diw2

Reply:
yes you need this

Reply:
"netnolimit" gives unrestricted network access - only do that for services that need that

Reply:
turns out i need do it - the service is unable to fetch keys to decode tokens without it